### PR TITLE
TLS config renegotiation setting (Issue #10871)

### PR DESCRIPTION
**Pull request by konstantinoskostis**
This Pull Request fixes the issue described at https://github.com/elastic/beats/issues/10871

### CODE REVIEW

**Comment by elasticcla** [comment]
Hi @konstantinoskostis, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in your Git [commit](https://github.com/elastic/beats/pull/12354.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

**Comment by ph** [comment]
@konstantinoskostis Can you sign the CLA?

**Comment by ph** [comment]
jenkins test this please

**Comment by konstantinoskostis** [comment]
@ph hello!
I have already signed the CLA ... but as i wrote in the issue
Not sure why the CLA signature is marked as erroneous. My email is correct, present in https://patch-diff.githubusercontent.com/raw/elastic/beats/pull/12354.patch and present on github!

Let me sign it one more time and see if it works.

**Comment by ph** [comment]
@konstantinoskostis Not all email are actually valid in the PR, like this one:

```
From 3c67cf822dc4bd8b5e3765ad01700471267acafd Mon Sep 17 00:00:00 2001
From: konstantinos kostis <konstantinos@MacBook-Pro.local>
Date: Wed, 29 May 2019 21:49:22 +0300
Subject: [PATCH 1/3] [tlscommon] Config: Fix yaml typo
```

**Comment by konstantinoskostis** [comment]
@ph sorry about that... I edited the commit to fix the author and now CLA check has gone green!

**Comment by ph** [comment]
jenkins test this please

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -54,6 +54,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Affecting all Beats*
 
+- Fix a yaml typo in libbeat/common/transport/tlscommon/config.go {issue}10871[10871], {pull}12354[12354]
```
Comment (kvch) [inline CHANGELOG.next.asciidoc]: I think one line should be enough to describe the change. As the Changelog is intended for users, mentioning `tls.Config` is not appropiate. I would reword this.

Comment (konstantinoskostis) [reply CHANGELOG.next.asciidoc]: @kvch your proposition sounds and looks better, thank you! I updated the entry accordingly!

**Comment by kvch** [comment]
Closes #10871
